cmake/toolchains/gnu4-toss_3_x86_64_ib-librom-dev.cmake

```diff
@@ -0,0 +1,22 @@
+# Toolchain for Lawrence Livermore National Laboratory TOSS3 machines
+# (e.g., Quartz), assuming the following dependencies:
+#
+# - MVAPICH2/2.3 with Intel 19.0.4 compilers
+# - Intel MKL
+# - HDF5 1.8.18
+
+# Use MPI compiler wrappers because it simplifies detection of MPI
+set(CMAKE_C_COMPILER mpicc)
```

Comment:
How about using the full path for the compilers so the toolchain file can work without doing "module load"?